cmd/recommend.go

```diff
@@ -16,6 +18,12 @@ var recommendCmd = &cobra.Command{
 	Short: "Recommend Policies",
 	Long:  `Recommend policies based on container image, k8s manifest or the actual runtime env`,
 	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(recommendOptions.Images) < 1 {
+			if err := cmd.Help(); err != nil {
+				return err
+			}
+			os.Exit(0)
```

Comment:
Hey @vishnusomank , will this show an error saying atleast one container image needs to be passed as an input?

Reply:
Hi @nyrahul it will not show an error message but instead shows the basic help for `karmor recommend`
I can add the error message if needed

Reply:
Added a proper error message and removed the help menu